Source/Scene/GroundPrimitive.js

```diff
@@ -599,6 +599,8 @@ define([
             boundingVolumes = groundPrimitive._boundingVolumes;
         } else if (frameState.mode !== SceneMode.SCENE3D && defined(groundPrimitive._boundingVolumes2D)) {
             boundingVolumes = groundPrimitive._boundingVolumes2D;
+        } else {
+            return;
```

Comment:
Can this else block ever actually get hit in normal execution?  Would it be better to throw a RuntimeError here instead of silently failing?

Reply:
@bagnell ?